Documentation/ApiOverview/Fluid/Syntax.rst

```diff
@@ -346,12 +346,19 @@ encapsulate the part in HTML comments:
 
 Note: This way the content will still be transferred to the browser! If
 you want to completely skip parts of your template, you can make use of
-the **f:comment** view helper. To disable parsing you best combine it
+the :ref:`t3viewhelper:typo3fluid-fluid-comment` view helper. To disable parsing you best combine it
 with CDATA tags:
 
+..  versionchanged:: 13.3
+    The content of the :ref:`t3viewhelper:typo3fluid-fluid-comment` is removed
+    before parsing. It is no longer necessary to combine it with CDATA tags
+    to disable parsing.
+
 ..  code-block:: html
     :caption: EXT:my_extension/Resources/Private/Templates/SomeTemplate.html
 
-    <f:comment><![CDATA[
-    This will be ignored by the Fluid parser and won't appear in the source code of the rendered template
-    ]]></f:comment>
+    <f:comment>
+        This will be ignored by the Fluid parser and wil not appear in
+        the source code of the rendered template
+        <x:someBrokenFluid>
```

Comment:
Do we really need to hint the user, that broken tags are ignored and don't throw an error now?
Looking at that at a user I think: "Why should I add some broken Fluid tag?"
It is a comment, I expect, that it is ignored whatever I write in it.

I opt for removing that `<x:someBrokenFluid>`

Reply:
As this is new and ppl have been used to not doing this I think yes it is important

Reply:
So people rely on using invalid syntax that throwed an error in the past in a comment? Strange.

Reply:
I would remove the broken fluid tag, but I also would write in the text above that ANY kind of fluid tags are not processed (anymore).

Reply:
I moved it to its own example